web/custom-plugins/forecast-of-response/src/components/Operations/OperationDetails.tsx

```diff
@@ -1,106 +1,132 @@
-                    <FlexWrapper>
-                      <StatusDot status={OperationStatus.IN_PROGRESS} />
-                      {phase.status}
-                    </FlexWrapper>
-                  </Table.Cell>
-                </Table.Row>
-              ))}
-          </Table.Body>
-        </Table>
-      </Card>
-    </>
+    <Card style={{ border: 'solid 1px', maxWidth: '1200px' }}>
+      <Card.Header>
+        <Card.HeaderTitle>
+          <Typography variant="h5">{operation.name}</Typography>
+          <Typography variant="body_short">
+            {operation.location.name}
+          </Typography>
+        </Card.HeaderTitle>
+      </Card.Header>
+      <div
+        style={{
+          display: 'flex',
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+        }}
+      >
+        <div style={{ display: 'flex', flexDirection: 'column' }}>
+          <FlexWrapper>
+            <Label label="Creator:" />
+            {operation.creator}
+          </FlexWrapper>
+          <FlexWrapper>
+            <Label label="Start:" />{' '}
+            {new Date(operation.start).toLocaleString(local)}
+          </FlexWrapper>
+          <FlexWrapper>
+            <Label label="End:" />{' '}
+            {new Date(operation.end).toLocaleString(local)}
+          </FlexWrapper>
+          <FlexWrapper>
+            <Label label="Status:" />
+            {operation.status}
+            <StatusDot status={operation.status} />
+          </FlexWrapper>
+        </div>
+        <LocationWrapper>
+          <Label label="Location:" /> {operation.location.name} (
+          {operation.location.lat},{operation.location.long})
+          <LocationOnMap location={operation.location} zoom={7} />
+        </LocationWrapper>
+      </div>
+      <Card.Actions>
+        <Button>Cancel</Button>
+        <Button>OK</Button>
+      </Card.Actions>
+      <Table density="comfortable" style={{ width: '100%' }}>
+        <Table.Caption>
+          <Typography variant="h3">Phases in operation</Typography>
+        </Table.Caption>
+        <Table.Head>
+          <Table.Row>
+            <Table.Cell>Phase</Table.Cell>
+            <Table.Cell>Started</Table.Cell>
+            <Table.Cell>Ended</Table.Cell>
+            <Table.Cell>Progress</Table.Cell>
+          </Table.Row>
+        </Table.Head>
+        <Table.Body>
+          {operation.phases.length &&
+            operation.phases.map((phase, index) => (
+              <Table.Row key={index}>
+                <Table.Cell>{phase.name}</Table.Cell>
+                <Table.Cell>
+                  {new Date(phase.start).toLocaleString(local)}
+                </Table.Cell>
+                <Table.Cell>
+                  {new Date(phase.end).toLocaleString(local)}
+                </Table.Cell>
+                <Table.Cell>
+                  <FlexWrapper>
+                    {phase.status}
+                    <StatusDot status={OperationStatus.ONGOING} />
```

Comment:
can we use phase.status for the StatusDot component?

Reply:
fixed!